### ISSUE TITLE
[Question - Couchbase] Android minimum SDK version

### ISSUE THREAD

**Issue by DimitrisRK** [triniwiz/nativescript-plugins#6]
I was curious if it's mandatory to use API 22 as minimum SDK version for couchbase plugin.

We happen to have users who still use android devices with lower API (mostly tablets). This also causes our apps not to run in few of our testing devices (minimum API 19).
I know we should move forward and also force few of our customers to get rid of their old equipment, but is there any future consideration for lower API support?

**Comment by triniwiz** [comment]
The minimum was set by the [Couchbase lib ](https://docs.couchbase.com/couchbase-lite/current/android/start/java-android-gs-prereqs.html)

**Comment by DimitrisRK** [comment]
> The minimum was set by the [Couchbase lib ](https://docs.couchbase.com/couchbase-lite/current/android/start/java-android-gs-prereqs.html) if it was up to me I would use 17 unless there is some new API I really need

Oh I see. Well, I believe I'll figure something out, but I also hope their team does not label more API levels as deprecated anytime soon. :D 
Thank you very much for this quick response!

**Comment by triniwiz** [comment]
They moved fast it was [announced](https://docs.couchbase.com/couchbase-lite/2.6/java.html#2-6-0) for 2.6 which was released sometime in August 2019 from the date on the [blog](https://blog.couchbase.com/announcing-couchbase-mobile-2-6/)

**Comment by DimitrisRK** [comment]
@triniwiz There have been answers in Couchbase forums for those who use lower API that they can try running latest version at their own risk. I also attempted to use 2.8.1 at API 19 and have no issues so far. Of course, there is no guarantee if it stops working in a week or a year.

The only thing I changed in my app to make it work is using an override inside AndroidManifest.xml (that's because plugin native code is compiled using API 22 as minimum)

Here is what I did, just for reference.
In AndroidManifest.xml, one has to add `xmlns:tools="http://schemas.android.com/tools"` in `manifest` tag and include the following rule as a child:
`<uses-sdk tools:overrideLibrary="com.github.triniwiz.couchbase"/> `
